components/match2/wikis/halo/match_summary.lua

```diff
@@ -147,11 +147,17 @@ function CustomMatchSummary._createMapRow(game)
 	local leftNode = mw.html.create('div')
 		:addClass('brkts-popup-spaced')
 		:node(CustomMatchSummary._addCheckmark(game.winner == 1))
-		:node(DisplayHelper.MapScore(game.scores[1], 1, game.resultType, game.walkover, game.winner))
+		:node(DisplayHelper.MapScore(
+			game.scores[1] and game.scores[1] .. (game.extradata.points1 and ' (' .. game.extradata.points1 .. ')' or ''),
```

Comment:
maybe extract to (local) function?

```lua
local getScoreValue = function(opponentIndex)
local score = game.scores[opponentIndex]
if not score then return end
local points = game.extradata['points' .. opponentIndex]
if not points then return score end
return score .. ' (' .. points .. ')
end
```

am on phone, please excuse the missing indents^^